test/extensions/filters/network/mysql_proxy/mysql_clogin_test.cc

```diff
@@ -553,6 +556,180 @@ TEST_F(MySQLCLoginTest, MySQLClientLogin41IncompleteAuthPluginName) {
   EXPECT_EQ(mysql_clogin_decode.getAuthPluginName(), "");
 }
 
+/*
+ * Negative Test the MYSQL Client Login 41 message parser:
+ * Incomplete length of connection attributions
+ */
+TEST_F(MySQLCLoginTest, MySQLClientLogin41IncompleteConnAttrLength) {
```

Comment:
There is lost of repeated code in next 5 tests. Consider maybe deriving another class from _MySQLCLoginTest_, for example _MySQL41LoginTest_ and move the repeated code to its methods.

Reply:
fixed.

Reply:
Looks much better!